app/renderer/js/utils/dnd-util.js

```diff
@@ -2,30 +2,168 @@
 
 const ConfigUtil = require(__dirname + '/config-util.js');
 
+const {ipcRenderer} = require('electron');
+
+let count = 0;
+let	dnd;
+let newSettings;
+let time = -1;
+function makeView() {
+	const dndOffTime = {
+		hr: new Date().getHours(),
+		min: new Date().getMinutes(),
+		carry: 0, // for dnd switch on after midnight 0 am
+		day: new Date().getDay()
+	};
+	const timeLeft = document.createElement('span');
+	const timeDisableDNDInput = document.createElement('div');
+	const actionContainer = document.getElementById('actions-container');
+	timeLeft.id = 'timeLeft';
+	actionContainer.prepend(timeLeft);
+	timeDisableDNDInput.className = 'dndInput';
+	for (let i = 1; i <= 4; i++) {
+		const opt = document.createElement('span');
+		if (i === 1) {
```

Comment:
These should be in a switch case. So many `if-else` is not preferred.

Reply:
sure. I will address in next commit